xayn-ai/src/coi/system.rs

```diff
@@ -52,15 +52,25 @@ impl CoiSystem {
         }
     }
 
+    /// Updates the view time of the positive coi closest to the embedding.
+    pub(crate) fn log_document_view_time(
+        &mut self,
+        cois: &mut Vec<PositiveCoi>,
+        embedding: &Embedding,
+        config: &Configuration,
+        viewed: Duration,
+    ) {
+        log_document_view_time(cois, embedding, config, viewed);
+    }
+
     /// Updates the positive coi closest to the embedding or creates a new one if it's too far away.
-    pub(crate) fn update_positive_coi(
+    pub(crate) fn log_positive_user_reaction(
         &mut self,
         cois: &mut Vec<PositiveCoi>,
         embedding: &Embedding,
         config: &Configuration,
         smbert: impl Fn(&str) -> Result<Embedding, Error>,
         candidates: &[String],
-        viewed: Duration,
     ) {
         update_positive_coi(
```

Comment:
since the name of the calling function changed, should we also change the name of the called function? even though this will unalign the names of the old implementation, i guess it's better to align the names of the new implementation.

imo it's also more descriptive, because the aim of the function is to log the reaction, updating the coi is just a consequence of that.

similarly for the negative case.

```suggestion
        log_positive_user_reaction(
```